common/common.go

```diff
@@ -56,6 +57,13 @@ const (
 	DefaultSystemConfigDir = "/usr/lib/rkt"
 )
 
+const (
+	FsMagicAUFS      = uint32(0x61756673)
+	FsMagicBTRFS     = uint32(0x1BADFACE)
+	FsMagicOverlayFS = uint32(0x794C7630)
+	FsMagicZFS       = uint32(0x2FC12FC1)
+)
```

Comment:
Several doubts on this:
 1. Where are these coming from? I was double checking with [statfs(2) manpage](http://man7.org/linux/man-pages/man2/statfs.2.html), but most of them are not there. Only BTRFS one is there but with a different value. If you have a single updated reference for those, can you add a comment there?
 2. Is the `uint32`(instead of `uint64`) choice due to typical value size? This introduces a type mismatch between magic constants and syscall interface, and an unchecked cast later. Can we get rid of that?
 3. BTRFS and OverlayFS constants are unused, can be removed.